src/bitmessagecurses/__init__.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=W0291 , W291
```

Comment:
I'd prefer you didn't do this, instead remove trailing spaces. Other than that it's ok.

Reply:
well, you'll be happy to hear that still trailing spaces are being shown as warnings. the statement does not seem to work. But it's helpful if other contributors feel annoyed by overly zelaous lint tests.

In particular, 80 char line length really is counterproductive in this day and age.

Reply:
Please just remove the first 2 lines even if it doesn't work. Then you can squash. Googling revealed that this can be done with

    # reset to upstream head
    git reset --soft 95e300d7cae5036d188ce9acc5bad6c91287dca1
    # create a single commit
    git commit